source/guide_matomo.rst

```diff
@@ -0,0 +1,107 @@
+.. highlight:: console
+.. author:: GodMod <godmod@eqdkp-plus.eu>
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/matomo.png
+      :align: center
+
+#########
+Matomo
+#########
+
+Matomo_ (former known as Piwik) is an open source website tracking tool (like Google Analytics) written in PHP and distributed under the GNU General Public License v3.0 licence.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * PHP_
+  * MySQL_
+  * domains_
+
+Prerequisites
+=============
+
+We're using PHP_ in the stable version 7.1:
+
+::
+
+ [isabell@stardust ~]$ uberspace tools version show php
+ Using 'PHP' version: '7.1'
+ [isabell@stardust ~]$
+
+.. include:: includes/my-print-defaults.rst
+
+If you want to use Matomo with your own domain you need to setup your domain first:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+If you want to install Matomo into a subfolder of your domain, create a new folder and navigate to it:
+::
+
+ [isabell@stardust ~]$ cd ~/html/
+ [isabell@stardust ~]$ mkdir matomo
+ [isabell@stardust ~]$ cd matomo/
+ [isabell@stardust matomo]$
+
+If you want to install matomo into your `document root`_, just navigate with ``cd`` to your `document root`_.
+
+Now download the latest version and extract it:
+
+.. note:: The link to the lastest version can be found at Matomo's `download page <https://matomo.org/download/>`_.
```

Comment:
The link is always the same so it makes no sense to send the user to the download page. No need to mention it then ☺️

Reply:
Removed.